packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -110,34 +110,32 @@ class EnrollVerbHandler extends AbstractVerbHandler {
   Future<void> _handleEnrollmentRequest(
       EnrollParams enrollParams,
       currentAtSign,
+      operation,
       Map<dynamic, dynamic> responseJson,
       InboundConnection atConnection) async {
+    String? otp = enrollParams.otp;
+    await _validateConnectionAuthAndOtp(otp, atConnection, operation);
     if (!atConnection.isRequestAllowed()) {
       throw AtThrottleLimitExceeded(
```

Comment:
Suggest move this "IF" condition also into the "_validateConnectionAuthAndOtp" and rename the method as more generic. For example "validateEnrollRequest"

Reply:
addressed in [6cd81fa](https://github.com/atsign-foundation/at_server/pull/1562/commits/6cd81faef234f9eb77d2743856046f94e27720ee)